nuget/publish.cmd

```diff
@@ -1 +1 @@
-@for %%f in (..\bin\*.nupkg) do @..\.nuget\NuGet.exe push %%f
\ No newline at end of file
+@for %%f in (..\bin\*.nupkg) do @..\packages\NuGet.CommandLine\tools\NuGet.exe push %%f
```

Comment:
if you setup the release process like in http://fsprojects.github.io/ProjectScaffold/release-process.html you don't need this extra publish.cmd